core/src/test/java/hudson/cli/handlers/ViewOptionHandlerTest.java

```diff
@@ -51,6 +51,8 @@
 import org.springframework.security.access.AccessDeniedException;
 import org.springframework.security.core.Authentication;
 
+import javax.annotation.Nonnull;
```

Comment:
To avoid regressing commit 819f80b12f:

```suggestion
import edu.umd.cs.findbugs.annotations.NonNull;
```

Reply:
Oops, that was misconfigured inside my IDE, thanks for pointing out.